eras/conway/impl/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 1.7.0.0
 
+* Add `ConwayEraGov`
```

Comment:
Should we also mention `constitutionGovStateL` ? 
Add `ConwayEraGov` with `constitutionGovStateL` or something like that?

Reply:
Done